Remove runtime section

trisberg opened the pull request.

trisberg committed:
Switch to use core runtime

markfisher commented on docs/v0.4/invokers/java.md:
````diff
@@ -115,20 +115,21 @@ The `--handler` option is the fully qualified name of the class that provides th
 ```
 
-This should create the resources needed for a Knative app. You can invoke the function using the `istio-ingressgateway` service. Assuming you haven't configured DNS for your function you can use the following command to find the IP address for the `istio-ingressgateway`:
+This should create the resources needed for a deploying the function. You can invoke the function using `kubectl proxy` command to access the service that was created.
````
s/proxy/port-forward/

trisberg replied:
duh!

trisberg committed:
Use port-forward instead of proxy

trisberg committed:
spacing

trisberg committed:
use terminal instead of bash for code snippets

scothis commented on docs/v0.4/invokers/java.md:
````diff
@@ -48,14 +48,14 @@ public class UpperApplication {
 You can build your function either from local source or from source committed to a GitHub repository. For local build use:
 
-```bash
+```terminal
````
`terminal` isn't a support language in [highlight.js](https://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases)

I've been using `sh`

trisberg replied:
I'd prefer something that doesn't highlight any keywords, like sh or bash highlights `function`

trisberg replied:
maybe just leave the language out although some editors complain - https://github.com/DavidAnson/markdownlint/blob/v0.16.0/doc/Rules.md#md040

scothis commented on docs/v0.4/invokers/java.md:
````diff
@@ -48,14 +48,14 @@ public class UpperApplication {
-```bash
-riff function create upper --git-repo https://github.com/<owner>/upper-boot-gradle.git --handler upper
+```terminal
+riff function create upper --git-repo https://github.com/<owner>/upper.git --handler upper
````
Can we link to something in projectriff-samples?

scothis commented on docs/v0.4/invokers/java.md:
```diff
@@ -107,28 +107,30 @@ public class Hello implements Function<String, String> {
 ## Deploying and invoking the function
 
-To deploy your function you need to select a runtime. The two options currently available are `core` and `knative` and we will select `knative`for this example:
+To deploy your function you need to select a runtime. The two options currently available are `core` and `knative` and we will select `core` for this example:
```
rm this whole section and link to the core runtime docs. There's nothing special about invoking a java function.

trisberg committed:
Switch to use sample uppercase app

- remove deploy/invoke section

- remove language for command line code snippets

trisberg committed:
Add note about no local builds on Windows
